packages/app/src/settings/project/CloudSettings.vue

```diff
@@ -1,5 +1,15 @@
 <template>
-  <ProjectId :gql="props.gql" />
+  <ProjectId
+    v-if="props.gql.currentProject?.projectId"
+    :gql="props.gql"
+  />
+  <section v-else>
```

Comment:
Do we need `<section>`? 

If possible let's try and keep the changes as minimal as possible - `<section>` seems a new introduction without and obvious reason why (unless I'm missing something).

Reply:
`<section>` was added here in order to prevent UI regression resulting from `children:pt-24px` className on parent element (shown in snippet below). Although this does not have to be `section` in particular, it could be another DOM element. `section` seamed appropriate, based on it's previous usage in `ProjectId` -> `SettingsSection`.
https://github.com/cypress-io/cypress/blob/823ffd0ca920c82170705d9abfa60592b8f3979a/packages/app/src/settings/SettingsCard.vue#L35-L40

Here's what UI looks like without `section`:
![Screenshot 2022-09-19 at 7 36 07 AM](https://user-images.githubusercontent.com/1501599/190961542-b1855775-0788-40cb-9cde-446e6edf2ccf.png)

WDYT? perhaps there's a more preferred resolution strategy for UI regression here?

Reply:
Right I see, I do not know much about a11y, is section appropriate? Probably fine for now... maybe @marktnoonan or @rachelruderman might have some input, resident a11y/semantic html experts.

We have Percy (UI snapshot service) so this will be caught by CI.